i2c/mcp9808_i2c/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(mcp9808_i2c
+        mcp9808_i2c.c
+        )
+
+# Pull in our (to be renamed) simple get you started dependencies
```

Comment:
A quick note here on this comment. In #132 this is being improved to be more descriptive, so would be good to get that in as well here, like this: https://github.com/raspberrypi/pico-examples/pull/132/files#diff-112d56e94629e0ec281a7eee133c1cc02557af71c1cd075466db5727ab7a42eaR5